react-native-libraries.json

```diff
@@ -13235,5 +13235,13 @@
     "images": ["https://i.ibb.co/WPfT696/Simulator-Screenshot-i-Phone-16-Pro-2024-12-24-at-11-07-48.png"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/ShivamJoker/react-native-quick-aws4",
+    "npmPkg": "react-native-quick-aws4",
+    "examples": ["https://github.com/ShivamJoker/react-native-quick-aws4?tab=readme-ov-file#example"],
```

Comment:
```suggestion
```

Package name link will link to the package github repository, so we expect full projects or Snacks in the `examples` array: 
* https://github.com/react-native-community/directory?tab=readme-ov-file#examples